shared/bid.js

```diff
@@ -1,3 +1,13 @@
+import { LISTING_STATUS, isOpen } from 'shared/listing'
+
 export function isAssetBid(bid, assetId, assetType) {
   return bid && bid.asset_type === assetType && bid.asset_id === assetId
 }
+
+export function fingerprintHasChanged(bid) {
```

Comment:
I think maybe `hasFingerprintChanged` is a bit more consistent with `isXXX => boolean`